src/hooks/useNotificationTypes.tsx

```diff
@@ -80,6 +88,13 @@ export const notificationTypes: NotificationTypeConfig[] = [
 
           switch (abacusNotificationType) {
             case 'order': {
+              const order = ordersById[id]?.[0];
+              const clientId: number | undefined = order?.clientId ?? undefined;
+              const localOrderExists =
+                clientId && localOrdersData.some((order) => order.clientId === clientId);
+
+              if (localOrderExists) return; // already handled by OrderStatusNotification
```

Comment:
will followup in notification preferences PR if this conflicts

Reply:
note: not sure if in other PR yet (will find out soon :D), but would it be possible to bundle the `Trading` in Notification Preferences to include all of `AbacusGenerated, TriggerOrder, and OrderStatus`?

Reply:
I was a little hesitant on this since order status seems a little more valuable than just the abacus generated Fills, and without it there may be little visual feedback with whether an order is being submitted. Will sync with product!

Reply:
will merge in the next one!